src/compiler/diagnosticMessages.json

```diff
@@ -3817,6 +3817,14 @@
         "category": "Error",
         "code": 4118
     },
+    "This member must have a leading JSDoc comment with an '@override' tag because it overrides a member in the base class '{0}'.": {

```

Comment:
I would remove 'leading' in the errors. It doesn't add anything, either for beginners or advanced Typescripters.

Reply:
Sure, updated